src/poetry/config/config.py

```diff
@@ -110,7 +110,7 @@ class Config:
         "virtualenvs": {
             "create": True,
             "in-project": None,
-            "path": os.path.join("{cache-dir}", "virtualenvs"),
+            "path": os.path.join("{cache-dir}", "virtualenvs"),  # noqa: PTH118
```

Comment:
Any reason why we can't replace this?

Reply:
from what I understand it would be `str(Path(...))` which doesn't look like a good use of resources, while `os.path` operates on strings all the way

Reply:
Add a comment maybe to avoid someone swapping it later.

But I reckon since it's not always a valid path (eg: template) it's okay.